firebase-firestore/firebase-firestore.gradle

```diff
@@ -96,6 +102,12 @@ tasks.withType(Test) {
     maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
```

Comment:
why is line 55 not enough?

Reply:
I have removed this task from the grade file. No other tasks except for Android will need this kotlinOption.